app/bundles/LeadBundle/Tests/Controller/ListControllerFunctionalTest.php

```diff
@@ -598,7 +598,7 @@ public function testSegmentFilterIcon(): void
 
     public function testUnpublishedSegmentDoesNotShowRebuildingLabel(): void
     {
-        // Create a segment that would normally show "Building" label
+        // Create a segment that would normally  show "Building" label
```

Comment:
This was committed probably by accident
```suggestion
        // Create a segment that would normally show "Building" label
```